Update README.md (#28)
versx/iPhoneController: README.md

`<string>/usr/local/lib/node_modules/pm2/bin/pm2 resurrect</string>`

2. From Mac desktop, Click the Apple, Open System Preferences  
![image](https://user-images.githubusercontent.com/3146205/124825495-b8217200-df41-11eb-8f9a-b0f154ff12f2.png)
3. Click Security and Privacy, click the lock to make changes and enter your password  
![image](https://user-images.githubusercontent.com/3146205/124825572-d38c7d00-df41-11eb-8185-64aff4b2b078.png)
4. On left, click full disk access then either click the plus and add the path from earlier OR open finder and navigate to the path and drag / drop into here  
![image](https://user-images.githubusercontent.com/3146205/124825746-0a629300-df42-11eb-87d2-d4b73f71daa0.png)
 
If you've already started iPhoneController with PM2 you'll now want to delete and relaunch after the above changes, this will now let PM2 access the files needed to run cfgutil.

## FAQ
Q. How do I get the profile ID?  
A. Run `security find-identity -p codesigning`  

Q. I'm receiveing `{"Command":"list","Output":{},"Type"::CommandOutput","Devices":[]}` when `cfgutil --format JSON list` is run.  
A. Reinstall Apple Configurator 2 automation tools  

> Q. I'm tring to reapply SAM but get error "failed to get ECID for device"?  
> A. You likely have useIosDeploy = true, you need this to be false to use cfgutil to support re-applying SAM 

Reviewer: ```suggestion
Q. I'm trying to reapply SAM but get error "failed to get ECID for device"?  
A. You likely have `useIosDeploy = true`, you need this to be "false" which uses `cfgutil` in order to support re-applying SAM.
```
Typo and minor adjustment to second sentence.
